junit-platform-console/src/main/java/org/junit/platform/console/tasks/ConsoleTestExecutor.java

```diff
@@ -101,6 +104,8 @@ private TestExecutionSummary executeTests(PrintWriter out, Optional<Path> report
 		Launcher launcher = launcherSupplier.get();
 		SummaryGeneratingListener summaryListener = registerListeners(out, reportsDir, launcher);
 
+		redirectStdStreams(outputOptions.getStdoutPath(), outputOptions.getStderrPath());
```

Comment:
The streams should be reset in a `finally` block after `launcher.execute` returns.